spec/features/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,39 @@
+feature "user deletes a review" do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:user1) { FactoryGirl.create(:user) }
```

Comment:
Either use `user1` and `user2`, or maybe more descriptive names? `author`, `other_user`, etc